ingest-app/docs/api.md

```diff
@@ -60,7 +60,7 @@ Content-Type is a standard HTTP header that specifies the content type of the bo
 Note: UMM JSON accepts an additional version parameter for both Content-Type: and Accept: headers. Like charset, it is appended with a semicolon (;). If no version is appended, the latest version is assumed.
 
 ```
-application/vnd.nasa.cmr.umm+json;version=1.1
+application/vnd.nasa.cmr.umm+json;version=1.14
```

Comment:
This should say something like `For example: application/vnd.nasa.cmr.umm+json;version=1.14 means UMM JSON format and version 1.14.`